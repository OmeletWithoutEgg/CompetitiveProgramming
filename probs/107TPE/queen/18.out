8150430.18 21242120.26
