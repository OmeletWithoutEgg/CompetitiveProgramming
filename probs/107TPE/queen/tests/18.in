436
-131694443 36257456
-43091832 40036138
86139533 -91627377
-138375351 61704535
-73590664 13467018
102079666 -17475287
-101014669 109607155
76581295 199628
-112866790 104822657
-43416720 39355374
-61443113 19105249
84047459 -2703676
94970608 -85511046
12280792 -49416768
-74561189 109283628
102028904 -17408518
85205494 -3283220
-76121517 12755845
-134019456 40814272
-38552665 57050033
-62155934 18657885
76439515 -95647659
26924294 -83086462
-122370809 24563560
15174518 -64548285
-40304289 47484300
86491372 -3971650
-125819062 94368609
-68160085 106960672
-137566674 52261911
-42318973 41772737
60805566 -97682014
-42172078 42124183
108362736 -67035427
-107233780 14922315
-134134475 41073371
-123946258 26089738
61575353 -97698353
17024798 -26382662
105826913 -72221806
-78572212 110259579
-131484855 86563532
12966505 -56128962
15826408 -66269536
61696388 -97699843
-58439877 21183829
-136892237 49133773
104838017 -73901797
41200851 -93055696
-127165360 29678764
99177705 -81414180
44167065 -94317904
-132608226 37914123
66095997 2149049
-114076409 104120683
-64366349 17373194
16456212 -67772796
-89731710 11248801
-44699856 36894176
-83353336 110977389
58039675 2119374
82677850 -93340195
-38453605 63986337
-118636621 21426045
-90216476 11264288
108230769 -67347225
62336843 2297004
13350786 -37275609
-47733056 32109658
111374522 -38379838
-77350458 12461597
98519748 -82121054
-38853262 68110876
104587555 -74304631
58527634 -97564066
23968155 -15540953
18672229 -23190581
37583516 -91194244
60097368 -97656503
111736416 -40545117
-42250186 80524548
102387509 -77520596
105119335 -21966936
44872259 -820463
-89290114 11238789
105624731 -22828135
12264289 -48837705
-125390636 94846455
111888425 -41689043
-90983539 11298417
97973574 -82687592
17634105 -25135335
-85633472 11305821
-137967111 54836693
-59641257 102147728
-100256511 109798882
92268381 -87690190
-105250759 108297397
111175107 -37384401
-73228810 108880375
-70418803 107893955
54958234 1762312
-63511898 17851930
51941512 1222591
33582424 -6738436
13818189 -60121581
87159559 -4348752
77028456 -95469428
-40826107 45775506
79903153 -94483432
93564729 -86683381
47602908 103158
-109136644 15743518
-127298365 92617942
-118223186 101345807
79014008 -596498
-132187170 85328623
-131676823 36226918
17721430 -70442335
111230680 -57754350
106519681 -70947712
-63893556 17635374
93984411 -86342498
89258767 -5624393
80027411 -94436357
13138127 -57077646
97938102 -82723775
16226893 -28164658
-46181515 34407140
91284914 -6995975
-91034373 11301096
92860438 -8167023
39956755 -2948613
-80219836 11900441
-115131061 103470872
99413582 -14251869
106053952 -71813862
-91238621 111148540
-88070042 11231407
64040898 2265044
18937460 -22724900
-124374394 26528434
-73012663 108811117
-94178404 110892828
33849689 -6552684
30165138 -86049646
72283255 -96686207
52102268 1256191
12429299 -51916859
13887997 -35015123
76689389 167159
106865723 -25129419
-112305622 105133191
-44633378 37013873
-110305527 106165592
84372880 -92543091
-43823089 83921775
106429141 -24286516
-41036800 77319240
47695178 131333
12324454 -50404586
-127440069 30019511
20486788 -75193218
-93553950 110961794
107294889 -69407417
94694977 -9658016
-71802098 108403047
40719955 -92829373
-81527645 11701903
-38379449 60800326
-128881137 31913855
77011454 68837
-56884558 100065863
-38528585 65113222
-118145547 101403453
34876897 -89543526
-63857501 17655643
-132835103 38349677
-49883817 29320764
-138185907 56856404
-38812489 67810725
-130702991 34611650
-75291588 12973272
-83890146 11432241
93070675 -87075159
-117264221 102041772
39979486 -2937296
-63288357 17980837
-45771523 35063263
34050942 -6414504
-116451027 19855559
101185729 -16332255
110731300 -59938130
-129855473 33309396
-38394211 62519253
108175112 -27928858
55346728 1818035
20494119 -20201494
-87411578 11239794
87013202 -91140853
-130728960 34652987
-42499467 41350898
46315285 -310520
-52211166 95755657
-101917960 13098782
-133432092 39548843
-42518092 81152953
-38680706 66727627
110913417 -59192445
-89380434 111220404
111610099 -39720451
111594684 -39625714
55263293 1806330
77746393 -164460
-39604186 72237472
98711183 -13487649
43635698 -1297587
44532486 -94458027
-104049908 108710760
111127205 -58246013
14168778 -61416436
-124245965 26395705
-80435751 11865220
-95867559 11794640
-108326696 15382522
19150264 -73046387
12311418 -45254007
-51641464 27312083
-46405659 34057749
12267176 -48958445
95291122 -10174568
31669370 -87259703
-38794986 54783401
-116563577 19932149
23646774 -79478418
24856982 -80894138
-65738216 105811381
36714344 -4716202
112034832 -52351682
91078380 -6849456
111593196 -55789222
-93659445 110950700
111905866 -53571060
-85628101 111154807
22453821 -77970924
111785042 -54516101
111367064 -57065221
-45557710 35414631
99339426 -14169670
43708212 -94137283
101224165 -79025829
-95297887 11711681
66360959 2127906
12254900 -47112198
-135188216 43658721
-46402616 34062449
19266047 -22163571
-118549309 21359822
-66346024 106114866
91616304 -88171467
58593805 2163116
-126679552 29090761
-126243257 28577867
98455878 -13217615
-134604041 80286064
-95105675 11685199
106482203 -71018947
-78258534 12265121
-134508621 80515916
111529683 -39238182
12746029 -54718420
103690567 -75681421
111619157 -39776664
-137824883 53816551
-118876649 21609660
-133822934 40380485
-117788789 20795553
-41435578 78448090
-61978521 18767670
-51534649 27428138
112132270 -44253313
46757532 -164100
-92976860 11442443
-55054362 98507289
64501798 -97652255
18919029 -22756888
-133309810 83164370
12621374 -41631727
109911473 -62819749
76775222 -95547034
-99142890 12403128
-78113843 110165677
-42850706 81901748
-138354387 59707091
110113950 -33239960
-114720343 103728224
112250943 -47486697
98492631 -13256241
-39114529 52677683
-119490020 22089470
99129101 -81467342
-41756679 79299493
-57775296 21689355
-38439960 63726917
48666509 416204
-38387957 60212756
-92153214 111087705
-122728665 24898599
111315799 -57330267
98549176 -13315829
53755860 -96975894
-55535971 98932285
-58492109 21144834
104341220 -20713149
-115432479 103278451
-129880800 89113866
16782368 -26904699
-81558168 110763232
112064547 -43384189
102368752 -17860019
86403736 -91482658
-137965912 54827402
23990030 -79890914
111715880 -55001299
33925280 -88905233
-134535838 80450684
-87899339 111228174
19347730 -73379259
105357556 -73037884
-134906081 42924716
108160885 -67509996
36617058 -4774146
91246189 -88437464
-137933574 67879170
-42860843 40536864
-38838771 54455171
86140732 -3779122
26839155 -83001254
63948183 2268278
13754549 -59870685
28376182 -84478852
66901473 -97486223
-66775245 16137909
-40943987 45417412
84224544 -2789881
-137445843 51622786
-66900732 106382917
107891234 -68123661
-129785729 89254857
73453404 -96431922
15729953 -66026514
-49362347 29960505
108694705 -66223734
-54655113 98146501
-71477664 14173132
35283834 -5598888
90520493 -6461452
-40329741 47395599
111870359 -41541758
13620516 -59323428
69489925 1770341
18183234 -71324859
58110941 -97531194
-63083011 18100613
17881663 -70753422
110856824 -59429549
107888965 -68128732
-98018510 110292186
-111940544 17130716
-38382620 60521827
50532109 -96310104
-135581205 77717413
-73132118 108849526
110015127 -32916871
-101301542 109531307
-118434612 101187638
-138279382 64362877
66048968 -97558500
-41104301 44944414
-91651881 111123137
-99615377 109951220
-51209146 94674537
-50643266 94034731
-111836257 17075152
-81211547 11746649
-104191896 13797267
-124902754 95375930
14339995 -33402703
15253660 -30638005
-138373582 61864206
-39050343 53056000
-78079937 12302370
-138373186 61894723
-113463120 104482243
78444896 -95008008
90594363 -88893663
-111887230 105358652
-79544767 110444089
-120403220 22833072
102923671 -76785001
64275557 2256088
12687289 -41115218
14723937 -63231584
20428681 -75104736
60997496 -97687198
-118953762 21669139
-71569502 14140252
-85216509 11330486
25903021 -13369345
-123041717 25197159
106487921 -24397748
-104855511 108437224
-89646025 111214370
-43014599 82258973
-138317129 58772155
-80338813 11880913
-72966530 108796195
-41692996 79134313
-39691102 49845159
14766615 -32044244
-137661167 69664314
84733733 -92363254
79288836 -695199
-117756608 20772165
12333566 -50568017
35559368 -89982172
79614965 -94591163
-44309181 84851948
109477851 -31281494
13079282 -56763926
53353785 -96904877
-104355408 13852095
-127856679 91912740
12449914 -43251970
-104547427 108543638
92345561 -87632138
