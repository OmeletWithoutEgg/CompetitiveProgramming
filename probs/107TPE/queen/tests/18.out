-13062268.44 6760122.29
